src/views/components/timeline/Timeline.css

```diff
@@ -0,0 +1,90 @@
+
```

Comment:
I'm concerned over the styling of this component, if it has a static CSS file which controls the styling then how do we restyle it across projects? Are we able to somehow leverage our layouts system to build the UI for this Timeline component and potentially ditch the static CSS?